.pre-commit-config.yaml

```diff
@@ -7,21 +7,21 @@ repos:
         language: system
         pass_filenames: false
         types: [python]
-        stages: [pre-commit]
+        stages: ['commit']
```

Comment:
I think this is a dependency problem (the SDK also had something similar).
Can we update the pre-commit dependency to match the SDK instead which should resolve this https://github.com/strands-agents/sdk-python/blob/1831bdb13df0c267c3a04d692d93688c71b975eb/pyproject.toml#L58

Reply:
updated the version resolving 🚀